feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature 1 branch.
```

Comment:
It is good that you have implemented these features.

Reply:
Thank you for the review, Abhishek